defaults/main.yml

```diff
@@ -30,25 +30,22 @@ elasticsearch:
     data: /var/lib/elasticsearch
     logs: /var/log/elasticsearch
     configuration: /etc/elasticsearch
+    pid: /var/run/elasticsearch
 
   plugins:
     - plugin_name: mobz/elasticsearch-head
-      plugin_dir:  head
+      plugin_dir: head
     - plugin_name: royrusso/elasticsearch-HQ
-      plugin_dir:  HQ
-    - plugin_name: elasticsearch/elasticsearch-lang-mvel/1.4.1
-      plugin_dir: lang-mvel
+      plugin_dir: hq
 
   plugin_config:
```

Comment:
Should we remove plugin_config and just move s3_backup to the parent hash? Eg.

```YAML
elasticsearch:
  s3_backup:
    enabled: false
    base_path: snapshots
    bucket: changeme
    region: changeme
```

In the future we could have a more configurable backup option but I think that can come later on.